-1 
Tested at: c9fc0365195dc9fe0b66e130752a5cd75714bc17
 I found an error when building: 
 
 <pre>Entering library rule at src/CondCore/Utilities/plugins
>> Compiling edm plugin /tmp/cmsbuild/workspace/ib-any-integration/CMSSW_7_6_X_2015-08-24-1100/src/CondCore/BeamSpotPlugins/src/plugin.cc 
>> Compiling  /tmp/cmsbuild/workspace/ib-any-integration/CMSSW_7_6_X_2015-08-24-1100/src/FWCore/Version/src/GetFileFormatVersion.cc 
>> Compiling  /tmp/cmsbuild/workspace/ib-any-integration/CMSSW_7_6_X_2015-08-24-1100/src/FWCore/Version/src/GetReleaseVersion.cc 
>> Compiling edm plugin /tmp/cmsbuild/workspace/ib-any-integration/CMSSW_7_6_X_2015-08-24-1100/src/CondCore/BeamSpotPlugins/plugins/BeamSpotObjects_toXML.cc 
/tmp/cmsbuild/workspace/ib-any-integration/CMSSW_7_6_X_2015-08-24-1100/src/CondCore/BeamSpotPlugins/plugins/BeamSpotObjects_toXML.cc:3:55: fatal error: CondCore/Utilities/interface/PayloadToXML.h: No such file or directory
 #include "CondCore/Utilities/interface/PayloadToXML.h"
                                                       ^
compilation terminated.
/tmp/cmsbuild/workspace/ib-any-integration/CMSSW_7_6_X_2015-08-24-1100/src/CondCore/BeamSpotPlugins/plugins/BeamSpotObjects_toXML.cc:3:55: fatal error: CondCore/Utilities/interface/PayloadToXML.h: No such file or directory
 #include "CondCore/Utilities/interface/PayloadToXML.h"
</pre> 
 you can see the results of the tests here: 
 https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-10842/7580/summary.html

--- a/CondCore/SiStripPlugins/plugins/SiStripObjects_toXML.cc
+++ b/CondCore/SiStripPlugins/plugins/SiStripObjects_toXML.cc
@@ -1,0 +1,26 @@
+
+#include "CondCore/Utilities/interface/PayloadToXML.h"
+#include "CondCore/Utilities/src/CondFormats.h"
+
+namespace { // Avoid cluttering the global namespace.
+
+  // converter methods
+  std::string SiStripLatency2xml( std::string const &payloadData, std::string const &payloadType ) { 
+    return cond::convertToXML<SiStripLatency> (payloadData, payloadType);
+  }
+
+  std::string SiStripConfObject2xml( const std::string &payloadData, const std::string &payloadType ) { 
+    return cond::convertToXML<SiStripConfObject> (payloadData, payloadType);
+  }
+
+} // end namespace
+
+
+BOOST_PYTHON_MODULE( pluginSiStripObjects_toXML )
+{
+    using namespace boost::python;
+    def ("SiStripLatency2xml"   , &SiStripLatency2xml);
+    def ("SiStripConfObject2xml", &SiStripConfObject2xml);
+
+}
+

--- a/CondCore/Utilities/interface/PayloadToXML.h
+++ b/CondCore/Utilities/interface/PayloadToXML.h
@@ -1,0 +1,42 @@
+
+#include <iostream>
+#include <string>
+#include <memory>
+
+#include <boost/python/class.hpp>
+#include <boost/python/module.hpp>
+#include <boost/python/init.hpp>
+#include <boost/python/def.hpp>
+#include <iostream>
+#include <string>
+#include <sstream>
+
+#include "boost/archive/xml_oarchive.hpp"
+#include "CondFormats/Serialization/interface/Serializable.h"
+#include "CondFormats/Serialization/interface/Archive.h"
+
+namespace cond { 
+
+  template<typename T>
+  std::string convertToXML( const std::string &payloadData, const std::string &payloadType ) { 
+
+      std::unique_ptr< T > payload;
+      std::stringbuf sdataBuf;
+      sdataBuf.pubsetbuf( const_cast<char *> ( payloadData.c_str() ), payloadData.size() );
+
+      std::istream inBuffer( &sdataBuf );
+      eos::portable_iarchive ia( inBuffer );
+      payload.reset( new T );
+      ia >> (*payload);
+
+      // now we have the object in memory, convert it to xml in a string and return it
+     
+      std::ostringstream outBuffer;
+      boost::archive::xml_oarchive xmlResult( outBuffer );
+      xmlResult << boost::serialization::make_nvp( "cmsCondPayload", *payload );
+
+      return outBuffer.str();
+
+  }
+} // end namespace cond
+

